src/zarr/testing/store.py

```diff
@@ -135,6 +154,26 @@ async def test_get(self, store: S, key: str, data: bytes, byte_range: ByteReques
         expected = data_buf[start:stop]
         assert_bytes_equal(observed, expected)
 
+    async def test_get_not_open(self, store_not_open: S) -> None:
```

Comment:
this is rather surprising -- I would expect that a non-open store would not support IO of any kind. what exactly does `open` mean? cc @jhamman

Reply:
Yes, this is also surprising to me!